packages/legend-application-pure-ide/src/stores/PureIDEStore.ts

```diff
@@ -880,6 +881,24 @@ export class PureIDEStore implements CommandRegistrar {
     );
   }
 
+  *debugging(command: { args: string[] }): GeneratorFn<void> {
```

Comment:
```suggestion
  *runDebugger(command: { args: string[] }): GeneratorFn<void> {
  // OR
  *executeDebugging(command: { args: string[] }): GeneratorFn<void> {
```

I think naming the function like either one above would be a little more clear